### PR TITLE
MGMT-10717: set OCP-4.10 as the default ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE

### PR DESCRIPTION
**Pull request by eranco74**
This should allow this branch to pass this release image to hypershift instead of letting hypershift defaulting to 4.11

### CODE REVIEW

**Comment by eranco74** [comment]
/hold
Let's check the capi test logs before merging

**Review by osherdp — APPROVED** [review]

**Pull request merged**
